src/server/repository/assessmentCycle/messageTopicUser/getOdpReviewStatus.ts

```diff
@@ -24,7 +24,7 @@ export const getOdpReviewStatus = async (
         from ${cycleSchema}.message m
         left join ${cycleSchema}.message_topic mt
           on mt.id = m.topic_id
-        where not m.deleted and mt.key like $1 || '-%'
+        where not m.deleted and (mt.key like 'odp-%-${odpId}-%' OR mt.key like '-%')
```

Comment:
why not using variable here like `$1` ?

Reply:
https://vitaly-t.github.io/pg-promise/formatting.html